src/Integration.Vsix.UnitTests/Suppression/LiveIssueFactoryTests.cs

```diff
@@ -267,6 +267,22 @@ public void Create_WhenIssueIsLineLevel_ReturnsExpectedIssue()
             result.LineHash.Should().Be("e1b4eea6db405a204a21bd5251c5385d");
         }
 
+        [TestMethod]
+        public void Create_WhenDiagnosticLocationIsNotInTheTextBuffer_ReturnsNull()
+        {
+            // Regression test for #1010
+            // Arrange & Act
+            var diagnostic = Diagnostic.Create(new DiagnosticDescriptor("id", "title", "message", "category",
+                DiagnosticSeverity.Hidden, true),
+                Location.Create("C:\\MySource.cs",
+                    new TextSpan(1, 1),
+                    new LinePositionSpan(new LinePosition(999, 100), new LinePosition(1000, 200))));
```

Comment:
999 is the magic number for non existing line, right?
How about we extract it to a local const for readability?
const int nonExistingLineNumber = 999;

Also, I think it's worth creating a helper method of CreateDiagnostic that receives only the necessary params. The other ones are just consts and they keep repeating in many tests.